spec/acceptance/foreman_basic_spec.rb

```diff
@@ -16,4 +16,17 @@
   describe package('foreman-telemetry') do
     it { is_expected.not_to be_installed }
   end
+
+  context 'GSSAPI auth enabled' do
+    it_behaves_like 'the foreman application' do
```

Comment:
I don't think you can use it this way since there's no application of it in the shared resource. So what I meant was:
```ruby
it_behaves_like 'an idempotent resource' do
  let(:manifest) do
    # ...
  end
end

it_behaves_like 'the foreman application'
```

Reply:
Thanks a lot for navigating here!